sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -231,7 +232,15 @@ def _create_pipeline(self, credential, **kwargs):
         elif credential is not None:
             raise TypeError("Unsupported credential: {}".format(credential))
 
-        config = kwargs.get("_configuration") or create_configuration(**kwargs)
+        config = kwargs.get("_configuration") or AzureTableConfiguration(
```

Comment:
I didn't see this whole code to create a pipeline, you don't really need to do that anymore

I would actually suggest taking a look at kv: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_shared/client_base.py#L46. We used to have this same code, where we would list all the policies, etc, but we changed it to this. Should get rid of. all of this creating a pipeline / transport / config before calling into thegeneratedclient